README.rst

```diff
@@ -88,7 +88,9 @@ Without ``setup.py``
 --------------------
 
 If you don't use ``setup.py`` (`it's easy to write one`_), you can create a
-``requirements.in`` file to declare the Flask dependency:
+``requirements.in`` **(make sure you don't have a requirement.txt file,
+otherwise, it might interfere with existing requirements.txt)** file to declare
```

Comment:
Could the repetitive word `requirements.txt` be avoided?

Reply:
`...otherwise, it might interfere.` would be good?